matlab/matlab-client/src/matlab/PAsolve.m

```diff
@@ -759,6 +780,13 @@ function initSolveConfig(solve_config,opt,sched)
 % Initialize Task Config Input Parameters
 function [input,main,taskFilesToClean,outVarFiles]=initParameters(solveid,NN,MM,Tasks,opt,taskFilesToClean,task_config,allfuncs,pa_dir,subdir,fs)
 
+    function answer = topath(list)
+        answer = list{1};
+        for z = 2:length(list)
```

Comment:
I am not familiar with the matlab syntax, so my question is probably stupid but why is the index starting at 2?